recipe/meta.yaml

```diff
@@ -271,3 +270,4 @@ extra:
     - hajapy
     - ocefpaf
     - hmaarrfk
+    - xhochy
```

Comment:
i guess you might want to add yourself to the main branch too.

Reply:
https://github.com/conda-forge/opencv-feedstock/pull/349